tests/helper/helper_generic.go

```diff
@@ -442,7 +444,6 @@ func SetProjectName() string {
 
 // RunTestSpecs defines a common way how test specs in test suite are executed
 func RunTestSpecs(t *testing.T, description string) {
-	os.Setenv("ODO_DISABLE_TELEMETRY", "true")
```

Comment:
Why have you removed this from here and added it to `CommonBeforeEach`?

Reply:
Moving it back